scripts/bump-sdk-in-connectors.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Bumps the SDK version in the built-in connectors.
+# The script assumes that the Conduit repo and the repos
+# for all built-in connectors are in the same directory.
+# Requires GitHub CLI.
+if [ $# -eq 0 ]
+then
+  echo "No arguments supplied"
+  exit 1
+fi
+
+if [ -z "$1" ]
+then
+  echo "Version is empty"
+  exit 1
+fi
+
+SDK_V=$1
+
+for conn in 'file' 'kafka' 'generator' 's3' 'postgres' 'log'
+do
+	cd ../conduit-connector-$conn
+
+	echo
+	echo "Working on conduit-connector-$conn"
```

Comment:
this should work right?
```suggestion
	echo "\nWorking on conduit-connector-$conn"
```

Reply:
It should, but I don't think we should block because of this 😉 Given that you approved the PR I guess you won't insist on this, so I'll resolve the comment and merge, to not keep the PR open for 1 more week.